docs/contributing.md

```diff
@@ -129,62 +129,50 @@ Prerequisites: pip3 install bandit
 </br>
 
 #### `ERROR`
-Log at this level errors of every ovms operation that failed 
+Log at this level errors related to the internal model server state.
 </br>
-
 Example:
-```
-    "Failed to create a temporary path PATH"
-```
 
+- exceptions thrown during accesing local files
 </br>
 
-#### `WARN`
-Log at this level everything that may lead to an error/need to be fix by administrator.
 </br>
 
+#### `WARN`
+Log at this level informations about conditions that may lead to an error or behavior unwanted or unexpected by the user.
+</br>
 Example:
-```
-    "Expected version directory to be in number format. Got:DIR_NAME"
-```
 
+- problems with configurations
+- version directories with invalid name
 </br>
 
 #### `INFO` 
-Log at this level all actions that are user-driven, or any others that changes state of application:
+Log at this level actions that changes state of application.
 </br>
+Example:
 
-- processing requests
 - (re)loading config/model
 - entry/exit points
 </br>
 
-Example:
-```
-    "Will add model: MODEL_NAME; version: MODEL_VERSION ..."
-```
-		
-</br>
-
 #### `DEBUG`
 Log at this level information about what happens in the program that may help during debbuging.
 </br>
 
 Example:
-```
-    "Getting default version for model:MODEL_NAME, DEFAULT_VERSION"
-```
 
```

Comment:
+ duration of specific operations (deserialization, serialization, inference etc.)